app/uk/gov/hmrc/pillar2/repositories/RegistrationCacheRepository.scala

```diff
@@ -18,57 +18,41 @@ package uk.gov.hmrc.pillar2.repositories
 
 import com.google.inject.Inject
 import com.mongodb.client.model.FindOneAndUpdateOptions
-import org.joda.time.{DateTime, DateTimeZone}
-import org.mongodb.scala.bson.{BsonBinary, BsonDocument}
+import org.joda.time.DateTime
+import org.joda.time.DateTimeZone
+import org.mongodb.scala.bson.BsonDocument
 import org.mongodb.scala.model._
 import play.api.Logging
 import play.api.libs.json._
+import uk.gov.hmrc.crypto.Crypted
+import uk.gov.hmrc.crypto.Decrypter
+import uk.gov.hmrc.crypto.Encrypter
+import uk.gov.hmrc.crypto.SymmetricCryptoFactory
+import uk.gov.hmrc.crypto.json.JsonEncryption
 import uk.gov.hmrc.mongo.MongoComponent
-import uk.gov.hmrc.mongo.play.json.formats.MongoBinaryFormats.{byteArrayReads, byteArrayWrites}
+import uk.gov.hmrc.mongo.play.json.Codecs
+import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
 import uk.gov.hmrc.mongo.play.json.formats.MongoJodaFormats
-import uk.gov.hmrc.mongo.play.json.{Codecs, PlayMongoRepository}
 import uk.gov.hmrc.pillar2.config.AppConfig
-import uk.gov.hmrc.pillar2.repositories.RegistrationDataEntry.RegistrationDataEntryFormats.expireAtKey
-import uk.gov.hmrc.pillar2.repositories.RegistrationDataEntry.{DataEntry, JsonDataEntry, RegistrationDataEntry, RegistrationDataEntryFormats}
+import uk.gov.hmrc.pillar2.repositories.RegistrationDataKeys.{expireAtKey, lastUpdatedKey}
```

Comment:
expireAtKey import not used and can be removed

Reply:
done